### PR TITLE
feat: add deprecation notice

### PR DESCRIPTION
**Pull request by emig**
waiting for https://github.com/sozialhelden/wheelmap/pull/431

### CODE REVIEW

**Review by Hoverbear — APPROVED** [review]
Just needs a link then good to go

**Comment by emig** [comment]
👍 welcome back

File: README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
+# DEPRECATED DEPRECATED DEPRECATED
```
Comment (Hoverbear) [inline README.md:1]: Why do this?

Comment (emig) [reply README.md:1]: mate high?  ¯\_(ツ)_/¯

**Pull request merged**